docs/platforms/apple/common/features/experimental-features.mdx

```diff
@@ -27,6 +27,7 @@ SentrySDK.start { options in
     options.enableTimeToFullDisplayTracing = true
     options.enableAppLaunchProfiling = true
     options.swiftAsyncStacktraces = true
+    options.profilesSampleRate = nil // enable continuous profiling mode
```

Comment:
`m`: would be good to add continuous profiling to the bullet points above (within the platformsection), something like 
`Enable <PlatformLink to="/profiling/#continuous-profiling">Continuous Profiling</PlatformLink> to get full coverage of your app's execution.`